client/src/components/cart/CartContent/CartContent.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { setSortedCart } from '../../../reducers/sortedCartReducer';
+import styles from './CartContent.module.scss';
+import shortid from 'shortid';
+import { useTranslation } from 'react-i18next';
+import CartDealer from '../CartDealer/CartDealer';
+
+function CartContent() {
+  const items = useSelector((state) => state.cart.items);
+  const sortedItems = useSelector((state) => state.sortedCart.items);
+  const dealers = Object.keys(sortedItems);
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+
+  const total = items.reduce((acc, cur) => {
+    return acc + parseFloat(cur.price) * cur.amount;
+  }, 0);
+
+  let ob = {};
+  for (let i of items) {
```

Comment:
Transfer this handling into a function, this operation is not about to  build a component structure, state, etc.